site/docs/4.10.0/admin/metrics.md

```diff
@@ -4,7 +4,7 @@ title: Metric collection
 
 BookKeeper enables metrics collection through a variety of [stats providers](#stats-providers).
 
-> For a full listing of available metrics, see the [Metrics](../../reference/metrics) reference doc.
+> For a full listing of available metrics, see the Metrics reference doc.
```

Comment:
Since the Metrics reference has no content at all, I suggest that we just remove this sentence.